sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -0,0 +1,591 @@
+package com.smartdevicelink.encoder;
+
+import android.annotation.TargetApi;
+import android.app.Presentation;
+import android.content.Context;
+import android.hardware.display.DisplayManager;
+import android.hardware.display.VirtualDisplay;
+import android.media.MediaCodec;
+import android.media.MediaCodecInfo;
+import android.media.MediaFormat;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.SystemClock;
+import android.util.DisplayMetrics;
+import android.util.Log;
+import android.view.Display;
+import android.view.MotionEvent;
+import android.view.Surface;
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+
+import com.smartdevicelink.proxy.rpc.OnTouchEvent;
+import com.smartdevicelink.proxy.rpc.ScreenParams;
+import com.smartdevicelink.proxy.rpc.TouchCoord;
+import com.smartdevicelink.proxy.rpc.TouchEvent;
+import com.smartdevicelink.proxy.rpc.enums.TouchType;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.reflect.Constructor;
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.FutureTask;
+
+
+public class VirtualDisplayEncoder {
+    private static final String TAG = "VirtualDisplayEncoder";
+    private static final int DENSITY = DisplayMetrics.DENSITY_HIGH;
```

Comment:
Should users be able to set this? If not, there's no need for this variable and we can use `DisplayMetrics.DENSITY_HIGH` when needed. If they should be able to we should qualify it correctly.